mne/chpi.py

```diff
@@ -149,6 +150,115 @@ def head_pos_to_trans_rot_t(quats):
     return translation, rotation, t
 
 
+def _apply_quat(quat, pts, move=True):
+    """Apply MaxFilter-formatted head position parameters to points."""
+    trans = np.concatenate(
+        (quat_to_rot(quat[:3]),
+         quat[3:][:, np.newaxis]), axis=1)
+
+    return(apply_trans(trans, pts, move=move))
+
+
+def extract_head_pos_ctf(raw, gof_limit=0.98):
```

Comment:
I'll change the name to `_calculate_head_pos_ctf` and make it private.

I like the idea of having a single method eventually. I had been hoping to evaluate the mne chpi fitting by feeding it CTF data, but it seems that ctf filters out the cHPI frequencies so that doesn't work.
I would suggest that we use this a little while and then make a new PR where we make calculate_head_positions a public method and at that time we think through how to unify things.